common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -146,6 +146,9 @@ sClientAssociationParams:
   bssid: sMacAddr
```

Comment:
This commit message headline (ad65f91c5bc80152c1fb29449e2795d4dfc2a58e tlvf: beerocks message common: add association frame to sClientAssoci…) is too long (>72 characteres).